data/campaigns/Under_the_Burning_Suns/utils/storytxt.cfg

```diff
@@ -4,7 +4,7 @@
     [story]
         [part]
             # the note should be changed to better reflect the current state of this campaign and faction
-            story= _ "Note: This campaign is probably not appropriate for beginners. It changes certain Wesnoth standards, such as the elves’ stats, the day/night cycle, recall costs and general approach to the difficulty levels. This campaign emphasizes role-playing elements and tends to have longer scenarios with objectives that change in the middle. For these reasons, we strongly suggest that you occasionally save your game mid-scenario, so you won’t lose all your progress if you get stuck and have to start over."
+            story= _ "Note: This campaign is probably not appropriate for beginners. It changes certain Wesnoth standards, such as the day/night cycle, recall costs and general approach to the difficulty levels, it is also making use of a custom faction of Quenoth elves as a replacement for standard elves. This campaign emphasizes role-playing elements and tends to have longer scenarios with objectives that change in the middle. For these reasons, we strongly suggest that you occasionally save your game mid-scenario, so you won’t lose all your progress if you get stuck and have to start over."
```

Comment:
I recommend removing this whole story `[part]` altogether, since the note in the campaign description makes it redundant.

Also, there is no easy or clear way for the player to back out of the campaign by the time they read this. They will have to skip the intro, wait for the scenario to load and then click through all the starting dialogue before they can access the interface to exit the campaign.

Reply:
Click what? You just press esc to skip everything.

Reply:
If players guess correctly that pressing `esc` skips through dialogue sequences then that will help speed up the process for them. But it is still a bit of a process to get out of the campaign beginning and they may not think to use this shortcut.

Reply:
Honestly I'm not really keen on modifying the campaign in any way other than balance at this point if I don't have to.

Reply:
An odd statement in a PR modifying UtBS with no balance implications.

Reply:
Eh.. I guess I also shouldnt be fix any issues and just leave them to others.